packages/admin/cms-admin/src/common/MasterMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import { Menu, MenuCollapsibleItem, MenuContext, MenuItemRouterLink, MenuItemRouterLinkProps, useWindowSize } from "@comet/admin";
+import * as React from "react";
+import { RouteProps, useRouteMatch } from "react-router-dom";
+
+export type MasterMenuItem = Omit<MenuItemRouterLinkProps, "to"> & {
+    route?: RouteProps;
+    to?: string;
+    subMenu?: MasterMenuItem[];
+};
+
+export type MasterMenuData = MasterMenuItem[];
```

Comment:
Should we call it MasterMenuConfig instead? Data is very generic, config is (a little) more specific IMO

Reply:
IMHO data is more appropriate since config is more focused on the behaviour of the menu.